pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -590,6 +604,11 @@ def plot_scene(self, frames_list, output_gif_path, norm_index=0, dy_rng=0, fps=3
             Frames per Second.
         show : bool, optional
             Either if show or only export gif.
+        view : str, optional
+           View to export. Options are ``"isometric"``, ``"xy"``, ``"xz"``, ``"yz"``.
+           The default is ``"isometric"``.
+        zoom : float, optional
+            Default zoom.
```

Comment:
We should indicate the default value in the description.

Reply:
Correct. I've modified some methods, this argument like many others was there. That's why I didn't touch it.